app/src/main/java/com/github/godspeed010/weblib/LibraryFragment.kt

```diff
@@ -118,11 +131,17 @@ class LibraryFragment : Fragment() {
         builder.setPositiveButton(android.R.string.ok,
             DialogInterface.OnClickListener { dialog, which ->
                 dialog.dismiss()
-                closeKeyboard()
+                hideKeyboard()
```

Comment:
I tried commenting out the `hideKeyboard()`, and the keyboard closes. 

Maybe it was never needed in the first place? I'm sure I would have realized if that was the case though... 

Actually, after some more thought, perhaps it's only needed when the keyboard is opened programatically.

Reply:
I looked into this, and it seemed like it is needed if the soft keyboard is forced open.